src/index.css

```diff
@@ -1,124 +1,182 @@
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  overflow: hidden;
 }
 
-.container {
-  margin: 120px auto;
-  max-width: 400px;
+.todo-list .todo-list-header {
   width: 100%;
+  height: 50px;
   background: #fff;
-  border-radius: 5px;
-  padding: 25px;
+  border-bottom: 1px solid rgba(0, 0, 0, 0.08);
 }
 
-header {
-  font-size: 25px;
-  font-weight: 600;
+.todo-list .todo-list-header h1 {
+  color: #545862;
+  font-size: 15px;
+  font-weight: 400;
+  padding: 15px;
+  line-height: 20px;
+  margin: 0;
+  width: 100%;
 }
 
-.container .inputField {
-  display: flex;
-  height: 45px;
+.todo-list .add-new-item {
   width: 100%;
-  margin: 20px 0;
+  padding: 0;
+  background: white;
+  border-bottom: 1px solid rgba(0, 0, 0, 0.08);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
 }
 
-.checkList #inputList {
-  padding: 10px;
-  width: 90%;
-  outline: none;
+.todo-list .add-new-item .add-new-input {
+  width: 100%;
+  padding: 0 40px 0 1rem;
+  line-height: 50px;
+  height: 50px;
   border: none;
-  font-size: 20px;
+  font-style: italic;
+  font-weight: 300;
+  font-size: 15px;
 }
 
-.inputField input {
-  width: 97%;
-  height: 100%;
-  font-size: 17px;
-  padding-left: 15px;
-  background: white;
-  outline: none;
-  border: none;
+.todo-list .add-new-item .add-new-input::placeholder {
+  font-family: "Open Sans", sans-serif !important;
+  font-weight: 300;
 }
 
-.add-btn {
-  cursor: pointer;
-  opacity: 0.5;
-  background: #fff;
+.todo-list .add-new-item .add-new-input:focus-visible {
   border: none;
+  outline: none;
 }
 
-.move-item {
-  background: white;
+.todo-list .add-new-item .add-new-btn {
+  width: 40px;
+  height: 50px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 12px;
+  color: #c5c5c5;
+  transform: rotate(90deg);
+  background: transparent;
   border: none;
-  opacity: 0.5;
+  outline: none;
+  cursor: pointer;
 }
 
-.move-item:hover {
-  opacity: 1;
-  cursor: pointer;
+.todo-list .add-new-item .add-new-btn:hover {
+  color: #545862;
+  transition: 0.3s;
 }
 
-.inputField button .active {
-  opacity: 1;
-  pointer-events: auto;
+.todo-list .todo-list-wrap {
+  list-style: none;
+  width: 100%;
 }
 
-.todoList {
+.todo-list .todo-list-wrap .list-item {
+  width: 100%;
+  padding: 0;
+  background: white;
+  border-bottom: 1px solid rgba(0, 0, 0, 0.08);
   display: flex;
-  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
 }
 
-.container .todoList {
-  max-height: 250px;
-  overflow-y: auto;
+.todo-list .todo-list-wrap .list-item .list-item-check-name {
+  display: flex;
+  align-items: center;
+  padding: 0 40px 0 1rem;
+  width: 100%;
 }
 
-.todoList li {
-  height: 45px;
-  background: #fff;
-  border-bottom: 1px solid rgba(0, 0, 0, 0.08);
-  margin-bottom: 8px;
-  cursor: default;
-  overflow: hidden;
+.todo-list .todo-list-wrap .list-item .list-item-check-name .description-input {
```

Comment:
Some nestings are too large, which creates a bit of confusion about what exactly is being changed. It would be better to create a more specific class for the particular case